src/_pytest/python.py

```diff
@@ -1428,15 +1428,15 @@ def write_fixture(fixture_def: fixtures.FixtureDef[object]) -> None:
         argname = fixture_def.argname
         if verbose <= 0 and argname.startswith("_"):
             return
-        if verbose > 0:
-            bestrel = get_best_relpath(fixture_def.func)
-            funcargspec = f"{argname} -- {bestrel}"
-        else:
-            funcargspec = argname
-        tw.line(funcargspec, green=True)
+        bestrel = get_best_relpath(fixture_def.func)
+        tw.write(f"{argname}", green=True)
```

Comment:
@The-Compiler, I noticed your suggestion to use `argname` in this line as a direct variable reference instead of using it as a formatted string. I am not able to find that comment to reference it.

I think that would be a good thing to do and keeps things consistent. Please let me know if this small change can go in as a separate PR.